run/image-processing/README.md

```diff
@@ -0,0 +1,35 @@
+# Cloud Run Pub/Sub Tutorial Sample
+
+This sample shows how to create a service that processes Pub/Sub messages.
+
+Use it with the [Cloud Pub/Sub with Cloud Run tutorial](http://cloud.google.com/run/docs/tutorials/pubsub).
+
+[![Run in Google Cloud][run_img]][run_link]
+
+[run_img]: https://storage.googleapis.com/cloudrun/button.svg
+[run_link]: https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_image=gcr.io/cloudrun/button&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&cloudshell_working_dir=run/cloudrun-pubsub
+
+## Build
+
+```
+docker build --tag pubsub-tutorial:java .
+```
+
+## Run
+
+```
+PORT=8080 && docker run --rm -p 8080:${PORT} -e PORT=${PORT} pubsub-tutorial:java
+```
+
+## Test
+
+```
+mvn clean verify
+```
+
+## Deploy
+
+```
+gcloud builds submit --tag gcr.io/${GOOGLE_CLOUD_PROJECT}/pubsub-tutorial
```

Comment:
I suspect it's the standard to use cloud build, but if you built it locally already, why not just do a push to gcr.io?

Reply:
Removing this in favor of tutorial explanation.